specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/ManagedDatabaseSensitivityLabels.json

```diff
@@ -854,6 +854,19 @@
             "name": "SensitivityLabelRank",
             "modelAsString": false
           }
+        },
+        "clientClassificationSource": {
```

Comment:
Do we have example files for this new enum property in DSMainDev?

Reply:
yes I have added example file